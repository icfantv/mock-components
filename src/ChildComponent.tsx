import * as React from 'react'
import { BindAll } from 'lodash-decorators'

@BindAll()
export class ChildComponent extends React.Component {
  public render(): React.ReactElement<ChildComponent> {
    return <div>Child Component</div>
  }
}
