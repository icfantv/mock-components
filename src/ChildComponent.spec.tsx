import * as React from 'react'
import { create } from 'react-test-renderer'
import { ChildComponent } from 'ChildComponent'

describe('test the <ChildComponent/> component', () => {
  test('it should render the child component', () => {
    const tree = create(<ChildComponent />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
