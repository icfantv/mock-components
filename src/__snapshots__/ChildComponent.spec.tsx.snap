// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test the <ChildComponent/> component it should render the child component 1`] = `
<div>
  Child Component
</div>
`;
